app/controllers/katello/api/v2/package_groups_controller.rb

```diff
@@ -3,6 +3,52 @@ class Api::V2::PackageGroupsController < Api::V2::ApiController
     apipie_concern_subst(:a_resource => N_("a package group"), :resource => "package_groups")
     include Katello::Concerns::Api::V2::RepositoryContentController
 
+    before_action :find_repository, :only => [:create, :destroy]
+
+    api :POST, "/package_group", N_("Create a package group")
+    param :id, String, :required => true, :desc => N_("package group Pulp ID")
```

Comment:
Yea, I don't really like it either, but Pulp requires it. We could require `name` and default `id` to `params[:metadata][:name].parameterize.underscore` ?